seimas/accounts/views.py

```diff
@@ -8,16 +8,20 @@
 import seimas.accounts.helpers.allauth as allauth_helpers
 import seimas.accounts.forms as accounts_forms
 from seimas.website.helpers import formrenderer
-
+#from allauth.account.auth_backends.AuthenticationBackend import AuthenticationBackend
 
 def login(request):
+    form1 = accounts_forms.LoginForm(request.POST)
+    if form1.is_valid():        
+          print (form1.cleaned_data['username'])
     openid_providers, form = allauth_helpers.get_openid_providers(request)
     if form:
         return allauth_helpers.openid_login(request, form)
     else:
         return render(request, 'accounts/login.html', {
             'auth_providers': allauth_helpers.get_auth_providers(request),
             'openid_providers': openid_providers,
+            'form': accounts_forms.LoginForm,
```

Comment:
WAT? Are you trying to pass class instance as a form to the template? I think you meant to pass `form1` here.